reference/customers.v3.yml

```diff
@@ -986,57 +986,57 @@ paths:
           type: number
           name: limit
           description: Items count per page. `limit=50`
-        - name: 'name:in'
+        - name: 'name'
           in: query
           required: false
           type: string
-          description: Filter items by first_name and last_name.
+          description: Filter items by customer attribute name `name=Date+Joined`
```

Comment:
```suggestion
          description: Filter items by customer attribute `name`. `name=Jane+Doe`
```

Reply:
`name` in this context refers to the name of the customer attribute, not the customer name. I used `Date+Joined` because it is an attribute I created for my sandbox store. We don't seem to actually explain what a customer attribute is, just that you can interact with them using the API. Should we expand on this section?

Reply:
@markcmurphy Great catch! Information on Customer Attributes is available in this [Customers and Subscribers](https://developer.bigcommerce.com/api-docs/store-management/customers#customers-api) overview article. Maybe we can link to this resource?